packages/protocol/contracts/common/OwnerUUPSUpgradable.sol

```diff
@@ -88,21 +88,23 @@ abstract contract OwnerUUPSUpgradable is UUPSUpgradeable, OwnableUpgradeable {
         _paused = _FALSE;
     }
 
-    function _inNonReentrant() internal view returns (bool) {
-        return _loadReentryLock() == _TRUE;
-    }
-
     // Stores the reentry lock
-    function _storeReentryLock(uint8 reentry) private {
-        assembly {
-            tstore(_REENTRY_SLOT, reentry)
-        }
+    function _storeReentryLock(uint8 reentry) internal virtual {
```

Comment:
so when you deploy on L1 for mainnet, maybe you want to use the tstore manually. @davidtaikocha